### PR TITLE
Bugfix/sumo 193330 monitor folder update

### PR DESCRIPTION
**Pull request by rmamilla-sumo**

### CODE REVIEW

**Review by tarunk2 — APPROVED** [review]
Some comments, LGTM

**Comment by sumovishal** [comment]
fixes #171

File: sumologic/resource_sumologic_monitors_library_monitor.go [hunk 1]
```diff
@@ -727,12 +727,13 @@ func resourceSumologicMonitorsLibraryMonitorRead(d *schema.ResourceData, meta in
 func resourceSumologicMonitorsLibraryMonitorUpdate(d *schema.ResourceData, meta interface{}) error {
 	c := meta.(*Client)
 	monitor := resourceToMonitorsLibraryMonitor(d)
-	if d.HasChange("parentId") {
-		// monitor.ParentID = d.Get("parentId").(string)
-		err := c.MoveMonitorsLibraryMonitor(monitor)
+
+	if d.HasChange("parent_id") {
```
[the remark this answers is not in the document]
Comment (rmamilla-sumo) [reply sumologic/resource_sumologic_monitors_library_monitor.go:731]: parent_id is the field we are accepting in terraform monitor resource.
eg:
resourceToMonitorsLibraryMonitor > https://github.com/SumoLogic/terraform-provider-sumologic/blob/master/sumologic/resource_sumologic_monitors_library_monitor.go#L1362

File: go.mod [hunk 2]
```diff
@@ -9,6 +9,7 @@ require (
 	github.com/go-errors/errors v1.4.0
 	github.com/go-test/deep v1.0.7 // indirect
 	github.com/hashicorp/go-hclog v0.16.2 // indirect
+	github.com/hashicorp/go-plugin v1.4.0 // indirect
```
Comment (sumovishal) [inline go.mod:12]: Why do we have changes here? Did you include some new module earlier?

**Review by sumovishal — COMMENTED** [review]
Can you pull the latest master? CHANGELOG has been updated since you last pulled.

**Review by dagould — APPROVED** [review]
lgtm -- I'd have released 2.17.0 already, but hit some (unrelated) snags, so we can include this.

File: go.mod (hunk 2 above)
Comment (rmamilla-sumo) [reply go.mod:39]: @sumovishal , didnt add anything manually. as part of go run , these are included.

Comment (sumovishal) [reply go.mod:39]: you can go ahead with the merge. We are just upgrading to a new version here. Should be fine.

**Pull request merged**